.github/workflows/ci-slow.yml

```diff
@@ -3,22 +3,36 @@ name: Slow CI
 on:
   workflow_dispatch:
   workflow_call:
-  # TODO: enable this with an issue_comment
-  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment
-  # TODO: disable this after tests show it works
-  push:
-    branches: [main]
-    paths-ignore: [docs/**, docker/**, '*', '!pyproject.toml', '**.md']
-  pull_request:
-    types: [opened, synchronize, ready_for_review]
-    paths-ignore: [docs/**, docker/**, '*', '!pyproject.toml', '**.md']
+  issue_comment:
+    types: [created]
+env:
+  TEAM_MEMBERS: '["@htahir1", "@hamza-zenml", "@bcdurak", "@baris-zenml", "@michael-zenml",
+    "@schustmi", "@strickvl", "@alex-zenml", "@alexej-zenml", "@AlexejPenner", "@stefannica",
+    "@wjayesh", "@safoinme", "@Cahllagerfeld", "@avishniakov", "atzin-escandia"]'
```

Comment:
```suggestion
    "@wjayesh", "@safoinme", "@Cahllagerfeld", "@avishniakov", "@atzin-escandia"]'
```
Maybe?

Reply:
Good catch. We have this file at `.github/teams.yml` but it was a pain to dynamically access it so I just hardcoded.